### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by jane-gorlova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,122 @@
...
+
+It requires the name of the image on which to run VS Code and references to the following 5 volumes: data, code, config, notebooks, and results. 
+These volumes will be mounted to /project/data, /project/modules, /project/config, /project/notebooks and /project/results respectively.
+
+### Quick example
+
+```
+jobs:
+  vscode:
+    action: gh:neuro-actions/vscode@master
```
Comment (anayden) [inline README.md]: `@v1.0.1` — this is the only repo where we have non `v1.0.0`. I've just added it to make the image `image` parameter optional

File: README.md [hunk 2]
```diff
@@ -1,3 +1,122 @@
...
+    args:
+      image: neuromation/base
```
Comment (anayden) [inline README.md]: In `v1.0.1` this argument is optional and will be equal to `neuromation/base` by default. We can remove it from the example.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,122 @@
...
+      volumes_config_remote: $[[ volumes.config.remote ]]
+      volumes_notebooks_remote: $[[ volumes.notebooks.remote ]]
+      volumes_results_remote: $[[ volumes.results.remote ]]
+```
+
+## Arguments
+
+### `image`
+
+The name of the image on which to run the VS Code instance.
```
Comment (anayden) [inline README.md]: I would add something along the lines: "Default is `neuromation/base`. If you use an image not derived from `neuromation/base` make sure it has the vscode server (https://github.com/cdr/code-server) installed"

File: README.md [hunk 4]
```diff
@@ -1,3 +1,122 @@
...
+### Example
+
+```
+args:
+	job_name: "vscode-job"
+```
+
+### `http_port`
+
+HTTP port to use for Jupyter. `"8080"` by default
```
Comment (anayden) [inline README.md]: It's not for Jupyter. It's for the VS Code itself.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,122 @@
...
+### Example
+
+```
+args:
+	http_port: 8282
+```
+
+### `http_auth`
+
+Whether to use HTTP authentication for Jupyter or not. `"True"` by default.
```
Comment (anayden) [inline README.md]: Same: Jupyter → VS Code Web UI

**Review by anayden — CHANGES_REQUESTED** [review]
Some changes are required, but it looks good overall!

File: README.md (hunk 1 above)
Comment (jane-gorlova) [reply README.md]: Fixed in the latest commit

File: README.md (hunk 2 above)
Comment (jane-gorlova) [reply README.md]: Fixed in the latest commit

File: README.md (hunk 3 above)
Comment (jane-gorlova) [reply README.md]: Fixed in the latest commit

File: README.md (hunk 4 above)
Comment (jane-gorlova) [reply README.md]: Fixed in the latest commit

File: README.md (hunk 5 above)
Comment (jane-gorlova) [reply README.md]: Fixed in the latest commit

**Pull request merged**